mappings/net/minecraft/world/gen/feature/RandomPatchFeatureConfig.mapping

```diff
@@ -0,0 +1,14 @@
+CLASS net/minecraft/class_4638 net/minecraft/world/gen/feature/RandomPatchFeatureConfig
+	CLASS class_4639 Builder
+	FIELD field_21237 stateProvider Lnet/minecraft/class_4651;
+	FIELD field_21238 blockPlacer Lnet/minecraft/class_4629;
+	FIELD field_21239 whitelist Ljava/util/Set;
+	FIELD field_21240 blacklist Ljava/util/Set;
+	FIELD field_21241 tries I
+	FIELD field_21242 xSpread I
```

Comment:
convention appears to be offsetX, not xOffset, same for the other two

Reply:
Yeah, favor convention over mojang names